Rakefile

```diff
@@ -1,12 +1,10 @@
 require 'puppetlabs_spec_helper/rake_tasks'
 require 'puppet-lint/tasks/puppet-lint'
 
-desc "Validate manifests, templates, and ruby files"
-task :validate do
-  Dir['spec/**/*.rb', 'lib/**/*.rb'].each do |ruby_file|
-    sh "ruby -c #{ruby_file}" unless ruby_file =~ /spec\/fixtures/
-  end
-  Dir['templates/**/*.erb'].each do |template|
-    sh "erb -P -x -T '-' #{template} | ruby -c"
-  end
-end
```

Comment:
Travis CI tests do run this code - and we like to have that check - as long as this is still done through the spec_helper I'm good with removing this code.